### PR TITLE
STY: Flake8 style fixes

### PR DESCRIPTION
**Pull request by Kai-Striega**

### CODE REVIEW

File: numpy_financial/_financial.py [hunk 1]
```diff
@@ -571,12 +572,18 @@ def pv(rate, nper, pmt, fv=0, when='end'):
...
 def _g_div_gp(r, n, p, x, y, w):
+    # Evaluate g(r_n)/g'(r_n), where g =
+    # fv + pv*(1+rate)**nper + pmt*(1+rate*when)/rate * ((1+rate)**nper - 1)
     t1 = (r+1)**n
     t2 = (r+1)**(n-1)
-    return ((y + t1*x + p*(t1 - 1)*(r*w + 1)/r) /
-                (n*t2*x - p*(t1 - 1)*(r*w + 1)/(r**2) + n*p*t2*(r*w + 1)/r +
-                 p*(t1 - 1)*w/r))
+    g = y + t1*x + p*(t1 - 1) * (r*w + 1) / r
+    gp = n*t2*x
```
Comment (WarrenWeckesser) [inline numpy_financial/_financial.py]: This line needs to have an opening parenthesis, e.g. `gp = (n*t2*x`, and the line that ends the expression gets the closing parenthesis.  Without those parentheses, the code simply assigns `n*t2*x` to `gp`, and the next three lines (which each happens to be a syntactically correct expression) are evaluated and then ignored.

Clearly the test coverage of this code is insufficient. 😮

File: doc/source/conf.py [hunk 2]
```diff
@@ -32,8 +32,8 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
-	'numpydoc',
-	'sphinx.ext.imgmath',
+    'numpydoc',
```
Comment (WarrenWeckesser) [inline doc/source/conf.py]: Thanks.  I upgraded my editor a while back, and I didn't realize that the default setting was no longer converting tabs to spaces.

File: numpy_financial/_financial.py (hunk 1 above)
Comment (Kai-Striega) [reply numpy_financial/_financial.py]: I've check this, the test calculates the result at the end of the period (i.e. w=0) so all the lines containing zero evaluate to 0 and the test passes. 

I'd prefer to get these moved across *as is* before making any behavioural changes, would it ok with you if I create an issue to increase the test coverage, using this as an example?

File: numpy_financial/_financial.py [hunk 3]
```diff
@@ -571,12 +572,19 @@ def pv(rate, nper, pmt, fv=0, when='end'):
...
 def _g_div_gp(r, n, p, x, y, w):
+    # Evaluate g(r_n)/g'(r_n), where g =
+    # fv + pv*(1+rate)**nper + pmt*(1+rate*when)/rate * ((1+rate)**nper - 1)
     t1 = (r+1)**n
     t2 = (r+1)**(n-1)
-    return ((y + t1*x + p*(t1 - 1)*(r*w + 1)/r) /
-                (n*t2*x - p*(t1 - 1)*(r*w + 1)/(r**2) + n*p*t2*(r*w + 1)/r +
-                 p*(t1 - 1)*w/r))
+    g = y + t1*x + p*(t1 - 1) * (r*w + 1) / r
+    gp = (
```
Comment (WarrenWeckesser) [inline numpy_financial/_financial.py]: Minor style quibble: Do you have a reason for the line break here?  The four terms are short enough that I think the style shown at https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator would look better.

File: numpy_financial/_financial.py (hunk 1 above)
Comment (WarrenWeckesser) [reply numpy_financial/_financial.py]: I didn't mean to suggest that tests were needed in *this* pull request!  

> would it ok with you if I create an issue to increase the test coverage, using this as an example?


Yes, definitely.  We might as well keep track of the issue, even if we don't improve it right away.

File: numpy_financial/_financial.py (hunk 3 above)
Comment (Kai-Striega) [reply numpy_financial/_financial.py]: I thought it looked nice to have the beginning of each expression aligned (this is also done in the example), while keeping the indentation a multiple of 4. The alternatives would either mis-align the top expression

```python
    gp = (n*t2*x
        - p*(t1 - 1) * (r*w + 1) / (r**2)
        + n*p*t2 * (r*w + 1) / r
        + p*(t1 - 1) * w/r)
```
or, if I add spaces to align that expression, contain 10 spaces, which then looks awkward with every other indentation being 4/8/12 spaces.

I'm not too concerned with the either style change, let me know if you want me to fix it.

Comment (WarrenWeckesser) [reply numpy_financial/_financial.py]: 10 spaces is fine.  That sort of alignment is pretty typical.

Comment (Kai-Striega) [reply numpy_financial/_financial.py]: Fixed. Is this fine to merge otherwise?

**Comment by WarrenWeckesser** [comment]
Thanks Kai.  I squashed the last couple commit and pushed to master.

**Pull request closed without merging**